src/Codeception/Module/REST.php

```diff
@@ -303,11 +303,33 @@ public function amBearerAuthenticated($accessToken)
     /**
      * Sends a POST request to given uri.
      *
-     * Parameters and files (as array of filenames) can be provided.
+     * Parameters and files can be provided separately.
      *
      * @param $url
      * @param array|\JsonSerializable $params
-     * @param array $files
+     * @param array $files A list of filenames or "mocks" of $_FILES (each entry being an array with the following
+     *                     keys: name, type, error, size, tmp_name (pointing to the real file path). Each key works
+     *                     as the "name" attribute of a file input field.
+     * @example
```

Comment:
Please add this example before `@annotations`
Please use the markdown code formatting as in other docblocks with examples. Current `@example` tag is not supported by our publishing tool and will be broken on a website